main.go

```diff
@@ -24,11 +25,13 @@ type Config struct {
 	AccessToken           stepconf.Secret `env:"access_token,required"`
 	CloneUrl              string          `env:"git_repo_url,required"`
 	VersionCodeFile       string          `env:"version_code_file,required"`
+	BitriseBranchName     string          `env:"BITRISE_GIT_BRANCH,required"`
```

Comment:
I suggest not to consume the env directly but use put a different meta name here then use `$BITRISE_GIT_BRANCH` as default in the step.yml

That also makes it more flexible

Reply:
Ok updated. I was thinking that maybe using the bitrise env was easier, as you can define it when scheduling/starting a build (as compared to creating a workflow and going into the  settings to change it).